### DISCUSSION TITLE
Which applications are accepted?

### DISCUSSION THREAD

**Discussion by beroal** [roddhjav/apparmor.d#506] [General]
Here I intend to query which applications are accepted in this project.

- Gimp
- TeXstudio

**Comment by roddhjav** [comment]
Gimp is already present ;)

Nowadays, I tend to accept every application (as long as the profile respect the guideline), with a few exception:
- No [generic tool](https://apparmor.pujol.io/development/structure/#programs-to-not-confine)
- IDE are *hard* to confine as they can run anything.
